src/Pester.Runtime.ps1

```diff
@@ -2266,6 +2273,7 @@ function PostProcess-ExecutedBlock {
                     $b.PassedCount += $child.PassedCount
                     $b.FailedCount += $child.FailedCount
                     $b.SkippedCount += $child.SkippedCount
+                    $b.InconclusiveCount += $child.OwnInconclusiveCount
```

Comment:
```suggestion
                    $b.InconclusiveCount += $child.InconclusiveCount
```

Typo. Would break `InconclusiveCount` in parent blocks. See related task in review comment.